clientd3d/d3drender.c

```diff
@@ -9205,6 +9205,8 @@ Bool D3DMaterialWorldPacket(d3d_render_packet_new *pPacket, d3d_render_cache_sys
 
 Bool D3DMaterialWorldDynamicChunk(d3d_render_chunk_new *pChunk)
 {
+	auto state = NULL;

```

Comment:
This is unexpected and leads to unnecessary if (state) checks later.  Try declaring state where it's first used.

Reply:
Got it, thanks.